packages/web3-eth-accounts/src/account.ts

```diff
@@ -0,0 +1,70 @@
+import { utils, getPublicKey } from 'ethereum-cryptography/secp256k1';
+import { toChecksumAddress, bytesToHex, sha3Raw, HexString } from 'web3-utils';
+import { InvalidPrivateKeyError, PrivateKeyLengthError } from './errors';
+
+// Will be added later
+export const encrypt = (): boolean => true;
+
+// Will be added later
+export const sign = (): boolean => true;
+
+// Will be added later
+export const signTransaction = (): boolean => true;
+
+/**
+ * Get account from private key
+ */
+export const privateKeyToAccount = (
+	privateKey: string | Buffer,
+): {
+	address: string;
+	privateKey: string;
+	signTransaction: () => boolean; // From 1.x
+	sign: () => boolean;
+	encrypt: () => boolean;
+} => {
+	if (!privateKey) {
+		throw new InvalidPrivateKeyError(privateKey);
+	}
+
+	const stringPrivateKey =
+		typeof privateKey === 'object' ? Buffer.from(privateKey).toString('hex') : privateKey;
```

Comment:
```suggestion
		typeof Buffer.isBuffer(privateKey) ? Buffer.from(privateKey).toString('hex') : privateKey;
```

I'm assuming this is going to work as expected

Reply:
If above change is added (i.e. `isBuffer` is added) then this would change to:

```typescript
typeof isBuffer(privateKey) ? Buffer.from(privateKey).toString('hex') : privateKey;
```